packages/access-client/src/stores/types.ts

```diff
@@ -5,8 +5,8 @@ import {
   DelegationMeta,
   SpaceMeta,
 } from '../types.js'
-import { RSASigner } from '@ucanto/principal/rsa'
 import { SignerArchive, DID } from '@ucanto/interface'
```

Comment:
```suggestion
import { SignerArchive, DID, DIDKey } from '@ucanto/interface'
```

We have this one now so you can type less